ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -558,6 +560,8 @@ class EditMode {
 
         editMode.active = true;
         editMode.isContextDetectionActive = true;
+
+        eventsOptions?.activate?.call(editMode);
```

Comment:
I'd do that with `addEvent`  / `fireEvent`. If users will like to listen to that event from out of the API, there should be such way. So here, should just be `fireEvent` called, and in the constructor the event should be registered.

Reply:
My first thought was the same, but I was wondering if adding the event to the editmode object makes sense. Im not convinced.

Reply:
I would certainly unify them. As I look at the first one `mount` it's done by `fireEvent` so we should stick with that 😉

Reply:
I'd add the event to the Board class.